backend/app/services/chatbot_service.py

```
@@ -371,6 +378,48 @@ async def upload_chatbot_files(self, user_id: ObjectId, chatbot_id: str, uploade
+                detector = DatabaseDetector()
+                storage_mgr = DatabaseStorageManager()
+
+                saved_db_info = []
+                for db_file in db_candidates:
+                    saved_path = storage_mgr.save_uploaded_file(chatbot_id, db_file.file, db_file.filename)
+                    detect = detector.detect_db_type(saved_path)
+                    saved_db_info.append({
+                        "original_name": db_file.filename,
+                        "saved_path": str(saved_path),
+                        "db_type": detect.get("db_type"),
+                        **({"error": detect["error"]} if detect.get("error") else {})
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)